src/pipelinedefinition.cpp

```diff
@@ -617,8 +618,7 @@ class NodeValidator {
                 return result;
             }
 
-            if (
-                (dependantNodeInfo.kind == NodeKind::DL || dependantNodeInfo.kind == NodeKind::CUSTOM) &&
+            if (dependantNodeInfo.kind != NodeKind::ENTRY &&
```

Comment:
Why this change? We allow checking connection metadata correctness for EXIT node, which has no defined input metadata.